packages/next/export/worker.ts

```diff
@@ -352,7 +352,8 @@ export default async function exportPage({
         if (serverless) {
           req.url += (req.url!.includes('?') ? '&' : '?') + 'amp=1'
           // @ts-ignore
-          ampHtml = (await renderMethod(req, res, 'export')).html
+          ampHtml = (await renderMethod(req, res, 'export', undefined, params))
```

Comment:
The fourth parameter here should match [this line](https://github.com/vercel/next.js/blob/ee0a4a7086eb0acddc6419230a375ea291043797/packages/next/export/worker.ts#L240) omitting ampPath, this can be updated in a follow-up PR as it addresses different cases. Good catch!